frontend/src/modules/batchactions/actions.js

```diff
@@ -33,6 +33,58 @@ export function checkSelection(
 }
 
 
+function _updateUI(
```

Comment:
There's no need to use the leading underscore here, because modules have a notion of private / public. If it's not `export`ed then it is private, so that's quite easy to figure out.